atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/KvsDeleteBenchmarks.java

```diff
@@ -36,6 +39,18 @@ private Object doDelete(RegeneratingTable<Multimap<Cell, Long>> table) {
         return table.getTableCells();
     }
 
+    private Object doDeleteRange(ConsecutiveNarrowTable table, int numBatches) {
+        Iterable<RangeRequest> rangeRequests;
```

Comment:
I can't find any prohibition in our [coding standards](https://github.com/palantir/gradle-baseline/blob/develop/docs/java-style-guide/readme.md), which makes only a passing reference to ternary operators.